influxdb3_server/src/http.rs

```diff
@@ -165,6 +166,22 @@ impl Error {
             data: Option<T>,
         }
         match self {
+            Self::WriteBuffer(WriteBufferError::CatalogUpdateError(
+                err @ (CatalogError::TooManyDbs
```

Comment:
I was not aware of the `@`... very cool!

Reply:
Yeah it's a little known trick, but nice when you want to match on the type but still have it bound to a variable. Another fun bit of Rust trivia is that the @ sigil is also a bit of a leftover from GC pointers back when Rust had a GC.